rpc_differ/rpc_differ.py

```diff
@@ -197,6 +197,15 @@ def make_rpc_report(repo_dir, old_commit, new_commit,
     rpc_repo_url = args.rpc_repo_url
     osa_differ.update_repo(repo_dir, rpc_repo_url, args.update)
 
+    # If either of the commits look like a ref, prepend
+    # origin to it so that it resolves properly when
+    # testing it as a commit.
+    ref_check = re.compile('^(?!origin/)pr/\d+/\w+$')
+    if ref_check.match(old_commit):
```

Comment:
I'd call `validate_commits(repo_dir, [old_commit])` in a try block then retry with origin/ prepended if that fails.